demo/build.gradle

```diff
@@ -19,7 +19,7 @@ android {
         versionCode 1
         versionName "1.0"
 
-        manifestPlaceholders = [downloadAuthority: "${applicationId}"]
+        manifestPlaceholders = [downloadAuthority: "${applicationId}.downloads"]
```

Comment:
this and https://github.com/novoda/download-manager/pull/13/files#diff-d18483ff140c15dd49ed3d88822df403R205 , 1 is the client and 1 is the library therefore the connection is not apparent. Here is lies a problem